product_docs/docs/mysql_data_adapter/2/06_features_of_mysql_fdw.mdx

```diff
@@ -83,4 +87,6 @@ MySQL Foreign Data Wrapper allows the cleanup of foreign tables in a single oper
 
 For more information, see [DROP EXTENSION](https://www.postgresql.org/docs/current/sql-dropextension.html).
 
+## Truncate table
 
+From version 2.9.0 and later, MySQL Foreign Data Wrapper supports the `TRUNCATE TABLE` command on the foreign tables. However the `CASCADE` option is not supported with `TRUNCATE TABLE` command.
```

Comment:
be consistent with capitalization and font treatment for SQL commands and options. I think uppercase should be sufficient but if code format treatment is used primarily go with that. This comment applies to the new content above.

Reply:
changes done to new content